.gitignore

```diff
@@ -1,2 +1,2 @@
-node_modules
-build
+coverage/
```

Comment:
can we continue to use `.coverage` like everywhere else to avoid needing to go back and change everything for no particular reason?

Reply:
Why not just use the default directory? Istanbul also outputs to `coverage` by default. For some reason we customized the output directory.